wiki/Installation-guide.md

```diff
@@ -1,114 +1,84 @@
 # Installation guide
 
-## Basic installation
+## Launcher (BETA)
 
-PC side:
+Launcher will allow you to manage old, current and new installations of ALVR streamer and allow to automatically install and upgrade to specific ALVR client version on headset
 
-* Install SteamVR, **launch it once** then close it. This is to make sure it sets the environment correctly for ALVR.
```

Comment:
As for my testing, on Windows, SteamVR doesn't need to be launched manually the first time, you can just use the button on the dashboard, but the first time will fail to connect.